### PR TITLE
[Breadth] blockchain commands

### PR DESCRIPTION
**Pull request by MyronFanQiu**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S170

File: src/blockchain/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Microsoft Azure CLI 'blockchain' Extension
+==========================================
+
+This package is for the 'blockchain' extension.
+i.e. 'az blockchain'
```
Comment (haroldrandom) [inline src/blockchain/README.md:5]: better to a detail desciption

File: src/blockchain/azext_blockchain/generated/_help.py [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+"""
+
+helps['blockchain member create'] = """
+    type: command
+    short-summary: Create a blockchain member.
+    parameters:
+      - name: --firewall-rules
+        short-summary: Firewall rules of the blockchian member.
+        long-summary: |
+            Expect value: KEY1=VALUE1 KEY2=VALUE2 ... , available KEYs are: rule-name, start-ip-address, end-ip-address.
```
Comment (haroldrandom) [inline src/blockchain/azext_blockchain/generated/_help.py:45]: This line seems shouldn't be here.

File: src/blockchain/azext_blockchain/generated/_help.py [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+"""
+
+helps['blockchain member update'] = """
+    type: command
+    short-summary: Update a blockchain member.
+    parameters:
+      - name: --firewall-rules
+        short-summary: Firewall rules of the blockchian member.
+        long-summary: |
+            Expect value: KEY1=VALUE1 KEY2=VALUE2 ... , available KEYs are: rule-name, start-ip-address, end-ip-address.
```
Comment (haroldrandom) [inline src/blockchain/azext_blockchain/generated/_help.py:63]: This line seems shouldn't be here.

File: src/blockchain/README.md (hunk 1 above)
Comment (MyronFanQiu) [reply src/blockchain/README.md:5]: Sure. Have not done that

File: src/blockchain/azext_blockchain/generated/commands.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        operations_tmpl='azext_blockchain.vendored_sdks.blockchain.operations._blockchain_member_operations#BlockchainM'
+        'emberOperations.{}',
+        client_factory=cf_member)
+    with self.command_group('blockchain member', blockchain_member, client_factory=cf_member,
+                            is_experimental=True) as g:
+        g.custom_command('list', 'blockchain_member_list')
+        g.custom_show_command('show', 'blockchain_member_show')
+        g.custom_command('create', 'blockchain_member_create', supports_no_wait=True)
+        g.custom_command('update', 'blockchain_member_update')
+        g.custom_command('delete', 'blockchain_member_delete', supports_no_wait=True)
```
Comment (jsntcy) [inline src/blockchain/azext_blockchain/generated/commands.py:27]: add confirm

File: src/blockchain/azext_blockchain/generated/_params.py [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+    with self.argument_context('blockchain transaction-node list-api-key') as c:
+        c.argument('blockchain_member_name', options_list=['--member-name'], help='Blockchain member name.')
+        c.argument('transaction_node_name', options_list=['--name', '-n'], help='Transaction node name.')
+        c.argument('resource_group_name', resource_group_name_type)
+
+    with self.argument_context('blockchain transaction-node regenerate-api-key') as c:
+        c.argument('blockchain_member_name', options_list=['--member-name'], help='Blockchain member name.')
+        c.argument('transaction_node_name', options_list=['--name', '-n'], help='Transaction node name.')
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('key_name', help='Gets or sets the API key name.')
```
Comment (jsntcy) [inline src/blockchain/azext_blockchain/generated/_params.py:130]: is it an enum?

**Pull request merged**